### ISSUE TITLE
Centralize or align exe pref path loading

### ISSUE THREAD

**Issue by cgchoffman** [Komodo/KomodoEdit#3271]
### Short Summary

Every language pref screen loads it's interpreters in a different way and most of them ignore things that are being done in underlying code.  This doesn't need to be the case.  If this can be centralize so it's not language specific that would be great but I don't know where that would live.

Otherwise they should all be simplified and unified.

NOTE: The returned list of paths already includes pref set by user and it is always first in the list.

**Comment by Naatan** [comment]
I agree it would be nice if it did this, but I don't see what we would gain from changing all this at this stage.

**Comment by cgchoffman** [comment]
Less chance of bugs in what is traditionally very buggy code.
